web/themes/custom/bevan/sass/components/_layout.scss

```diff
@@ -11,8 +11,9 @@
 }
 
 @media only screen and (max-width : 768px) {
-	.contextual-region {
+	.govuk-table {
```

Comment:
I had a quick look to see if changing a table to display block doesn't anything weird and aparently [it does!](https://developer.paciellogroup.com/blog/2018/03/short-note-on-what-css-display-properties-do-to-table-semantics/)

I think we could try using the .views-element-container wrapper class for this instead?